db/migrate/20241122061430_change_status_in_todos.rb

```diff
@@ -0,0 +1,12 @@
+class ChangeStatusInTodos < ActiveRecord::Migration[7.0]
+  
+  def up
+    change_column :todos, :status, 'integer USING CAST(status AS integer)', default: 0, null: false
+  end
+
```

Comment:
what about the existing data if any
like some todo are already in pending status in sting formate